src/components/banner/Banner.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "components/banner/banner.css";
+
+const Banner = () => {
+  return (
+    <>
```

Comment:
You can remove this since, `div` is the only child it has

Reply:
will remove it, thanks for the feedback.